README.rst

```diff
@@ -11,7 +11,7 @@ DragNDoc is JS application to help splitting and ordering document pages in sub
 Try it
 ------
 
-Small PDF splitter application based on DragNDoc splitpdfapp.com http://splitpdfapp.com 
+Small PDF splitter application based on DragNDoc splitpdfapp.com http://splitpdfapp.com
```

Comment:
My editor removed a trailing space character on his own. I don't know a thing about this url, the project it is supposed to link to...